unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/SectionController/MenuSections/SectionProjectScenesController.cs

```diff
@@ -0,0 +1,87 @@
+﻿using DCL.Helpers;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+internal class SectionProjectScenesController : SectionBase, IProjectSceneListener
+{
+    public override ISectionSearchHandler searchHandler => sceneSearchHandler;
+
+    private readonly SectionProjectScenesView view;
+
+    private readonly SceneSearchHandler sceneSearchHandler = new SceneSearchHandler();
+    private Dictionary<string, SceneCardView> scenesViews;
+
+    public SectionProjectScenesController()
+    {
+        var prefab =
+            Resources.Load<SectionProjectScenesView>("BuilderProjectsPanelMenuSections/SectionProjectScenesView");
+        view = Object.Instantiate(prefab);
+
+        view.scrollRect.onValueChanged.AddListener((value) => RequestHideContextMenu());
```

Comment:
Not a big fan of the `Controller` subscribing to events from the view's fields. maybe we can wrap it in an event in the view.

Reply:
+1